Source/Core/InputCommon/ControllerInterface/SDL/SDL.cpp

```diff
@@ -435,8 +438,8 @@ InputBackend::InputBackend(ControllerInterface* controller_interface)
   // We want buttons to come in as positions, not labels
   SDL_SetHint(SDL_HINT_GAMECONTROLLER_USE_BUTTON_LABELS, "0");
 
-  // Disable DualSense Player LEDs; We already colorize the Primary LED
-  SDL_SetHint(SDL_HINT_JOYSTICK_HIDAPI_PS5_PLAYER_LED, "0");
+  m_config_changed_callback_id = Config::AddConfigChangedCallback([this] { SetPS5PlayerLED(); });
```

Comment:
The callback-id can just be a local variable. You can `RemoveConfigChangedCallback` within the `quit_guard`, just above `SDL_Quit`.

Reply:
Unless I'm misunderstanding what you mean, this won't work since the quit guard is within the hotplug thread.
> an enclosing-function local variable cannot be referenced in a lambda body unless it is in the capture list

Reply:
Resolved; Added to thread & quit guard's lambdas